### ISSUE TITLE
No results RSS feed

### ISSUE THREAD

**Issue by RM57380** [jfurrow/flood#319]
When i add rss feed with their URLS, after clicking on 'Adding', '0 matching' is write beside the label of the rss feed.

**Comment by rlko** [comment]
Have you tried to restart flood? See #301

**Comment by RM57380** [comment]
Yes, i already try and it's the same thing.
Even without regex rules on the rss feed.

**Comment by wopian** [comment]
If a feed has no regex rules it wont match anything. `[\s\S]*` will match everything

**Comment by RM57380** [comment]
Yes, but in the details of the feed, we must see the number of items presents and here i've 0, on ruTorrent it's working

**Comment by rlko** [comment]
On Flood there's none items list of RSS feeds. The 'match' you see in "Torrent Feeds" refers to which download rules matched in the feed.
Until it's fixed, for each new download rules you have to restart Flood.

I also came from rutorrent, and I also found this confusing at the start.

**Comment by RM57380** [comment]
Ah okay, thanks for the help. It's a little bit an frustration at the beginning.

**Comment by jfurrow** [comment]
Thanks for the issue, and sorry for the confusion! I'll repurpose this issue to try to improve the instructional text for this feature.

**Comment by nVitius** [comment]
We could add a warning label under the Match Pattern field that shows if it's empty. Something like "empty Match Pattern will return no results". Or we could just make it a required field, since there's probably no use-case for adding a feed that never matches... We could also just default it, and keep it under an "advanced settings" area.

**Comment by noraj1337** [comment]
It is also not possible to edit or fully see all details of an existing rule.

**Comment by bbashy** [comment]
So we can't use full regex like `/^debian-*amd64*/i`, just the bit in between? `^debian-*amd64*`
